### PR TITLE
[ISSUE-912] Refactor csi logs

### LINKED ISSUE #912: Refactor csi logs
**Is your feature request related to a problem? Please describe.**
Need to reduce the logs on csi.
For example, it is less common to display a log about the status of partitions on a node:
```
Jul 19 07:39:21.134 [INFO] [VolumeManager] [discoverVolumeCRs] Drive with path /dev/sdo, SN PHYG13310274960CGN doesn't have filesystem, partition table, partitions or PV.
Jul 19 07:39:54.125 [INFO] [VolumeManager] [discoverVolumeCRs] Drive with path /dev/sdn, SN BTYG13900CX7960CGN doesn't have filesystem, partition table, partitions or PV.
Jul 19 07:39:54.134 [INFO] [VolumeManager] [discoverVolumeCRs] Drive with path /dev/sdo, SN PHYG13310274960CGN doesn't have filesystem, partition table, partitions or PV.
Jul 19 07:40:27.456 [INFO] [VolumeManager] [discoverVolumeCRs] Drive with path /dev/sdn, SN BTYG13900CX7960CGN doesn't have filesystem, partition table, partitions or PV.
```  
and print log about disk status only if something changed in drivemgr:
```
Jul 19 09:13:38.979 [INFO] [HALManager] [VID:"ATA" PID:"ST8000NM012A-2KE" SerialNumber:"WSD5HCMS" Health:"GOOD" Type:"HDD" Size:8001000000000 Path:"/dev/sdb" Slot:"12" Firmware:"CALD" ] Drive 0
Jul 19 09:13:38.979 [INFO] [HALManager] [VID:"ATA" PID:"ST8000NM012A-2KE" SerialNumber:"WSD5JC2S" Health:"GOOD" Type:"HDD" Size:8001000000000 Path:"/dev/sdc" Slot:"13" Firmware:"CALD" ] Drive 1
Jul 19 09:13:38.979 [INFO] [HALManager] [VID:"ATA" PID:"ST8000NM012A-2KE" SerialNumber:"WSD5JME2" Health:"GOOD" Type:"HDD" Size:8001000000000 Path:"/dev/sdd" Slot:"14" Firmware:"CALD" ] Drive 2
Jul 19 09:13:38.979 [INFO] [HALManager] [VID:"ATA" PID:"ST8000NM012A-2KE" SerialNumber:"WSD5F5RJ" Health:"GOOD" Type:"HDD" Size:8001000000000 Path:"/dev/sde" Slot:"15" Firmware:"CALD" ] Drive 3
Jul 19 09:13:38.979 [INFO] [HALManager] [VID:"ATA" PID:"ST8000NM012A-2KE" SerialNumber:"WSD5HJQ4" Health:"GOOD" Type:"HDD" Size:8001000000000 Path:"/dev/sdf" Slot:"16" Firmware:"CALD" ] Drive 4
Jul 19 09:13:38.979 [INFO] [HALManager] [VID:"ATA" PID:"ST8000NM012A-2KE" SerialNumber:"WSD5F45D" Health:"GOOD" Type:"HDD" Size:8001000000000 Path:"/dev/sdg" Slot:"17" Firmware:"CALD" ] Drive 5
Jul 19 09:13:38.979 [INFO] [HALManager] [VID:"ATA" PID:"ST8000NM012A-2KE" SerialNumber:"WSD5EESQ" Health:"GOOD" Type:"HDD" Size:8001000000000 Path:"/dev/sdh" Slot:"18" Firmware:"CALD" ] Drive 6
Jul 19 09:13:38.979 [INFO] [HALManager] [VID:"ATA" PID:"ST8000NM012A-2KE" SerialNumber:"WSD5EF04" Health:"GOOD" Type:"HDD" Size:8001000000000 Path:"/dev/sdi" Slot:"19" Firmware:"CALD" ] Drive 7
Jul 19 09:13:38.979 [INFO] [HALManager] [VID:"ATA" PID:"ST8000NM012A-2KE" SerialNumber:"WSD5EGSM" Health:"GOOD" Type:"HDD" Size:8001000000000 Path:"/dev/sdj" Slot:"20" Firmware:"CALD" ] Drive 8
Jul 19 09:13:38.979 [INFO] [HALManager] [VID:"ATA" PID:"ST8000NM012A-2KE" SerialNumber:"WSD5FQ6Y" Health:"GOOD" Type:"HDD" Size:8001000000000 Path:"/dev/sdk" Slot:"21" Firmware:"CALD" ] Drive 9
Jul 19 09:13:38.979 [INFO] [HALManager] [VID:"ATA" PID:"ST8000NM012A-2KE" SerialNumber:"WSD5EGGQ" Health:"GOOD" Type:"HDD" Size:8001000000000 Path:"/dev/sdl" Slot:"22" Firmware:"CALD" ] Drive 10
Jul 19 09:13:38.979 [INFO] [HALManager] [VID:"ATA" PID:"ST8000NM012A-2KE" SerialNumber:"WSD5E9VV" Health:"GOOD" Type:"HDD" Size:8001000000000 Path:"/dev/sdm" Slot:"23" Firmware:"CALD" ] Drive 11
Jul 19 09:13:38.979 [INFO] [HALManager] [VID:"ATA" PID:"SSDSC2KG960G8R" SerialNumber:"BTYG13900CX7960CGN" Health:"GOOD" Type:"SSD" Size:960000000000 Path:"/dev/sdn" Slot:"24" Firmware:"DL69" ] Drive 12
Jul 19 09:13:38.979 [INFO] [HALManager] [VID:"ATA" PID:"SSDSC2KG960G8R" SerialNumber:"PHYG13310274960CGN" Health:"GOOD" Type:"SSD" Size:960000000000 Path:"/dev/sdo" Slot:"0" Firmware:"DL69" ] Drive 13
Jul 19 09:13:38.979 [INFO] [HALManager] [VID:"ATA" PID:"MTFDDAV480TDS" SerialNumber:"2142322096AB" Health:"GOOD" Type:"SSD" Size:480000000000 Path:"/dev/sda" Slot:"0" Firmware:"J004" ] Drive 14
```

### PR DESCRIPTION
**Pull request by AnnaDutova**
Signed-off-by: Dutova <Ann.Dutova@Dell.com>

### Resolves #912

The aim was to reduce the number of repetitive logs from VolumeManager. I decided to save the last log, and if the log changes, display a new message. Thus, the info will contain only relevant information.

## PR checklist
- [x] Add link to the issue
- [x] Choose Project
- [x] Choose PR label
- [ ] New unit tests added
- [ ] Modified code has meaningful comments
- [ ] All TODOs are linked with the issues
- [ ] All comments are resolved

## Testing
Launched the cluster with csi and check log  
```

Aug  1 06:38:59.802 [DEBU] [VolumeManager] [discoverVolumeCRs] Drive with path /dev/loop23, SN LOOPBACK1568310096 doesn't have filesystem, partition table, partitions or PV.
Aug  1 06:38:59.806 [DEBU] [Executor] [lsblk /dev/loop24 --output FSTYPE --noheadings] [3.934148ms] [3934148] stdout: 

Aug  1 06:38:59.815 [DEBU] [Executor] [fdisk --list /dev/loop24] [9.230775ms] [9230775] stdout: Disk /dev/loop24: 101 MiB, 105906176 bytes, 206848 sectors
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 512 bytes / 512 bytes

Aug  1 06:38:59.820 [DEBU] [VolumeManager] [discoverVolumeCRs] Drive with path /dev/loop24, SN LOOPBACK2770247236 doesn't have filesystem, partition table, partitions or PV.
Aug  1 06:38:59.820 [DEBU] [LivenessCheckHelper] updated with OK
Aug  1 06:39:08.422 [DEBU] [LivenessCheckHelper] Check: OK

```

### CODE REVIEW

**Review by rkiselenko — APPROVED** [review]

File: pkg/node/volumemgr.go [hunk 1]
```diff
@@ -814,6 +824,13 @@ func (m *VolumeManager) discoverDataOnDrives() error {
 	return nil
 }
 
+func (m *VolumeManager) isLogChange(logMessage, driveID string) bool {
```
Comment (mishoyama) [inline pkg/node/volumemgr.go:827]: I don't think this is a good idea to store log and compare it

**Review by mishoyama — CHANGES_REQUESTED** [review]

File: pkg/node/volumemgr.go (hunk 1 above)
Comment (AnnaDutova) [reply pkg/node/volumemgr.go]: I rolled back the changes and converted this message to debug. Also this message will be displayed when the `kubectl logs` is called.

File: pkg/node/volumemgr.go [hunk 2]
```diff
@@ -805,7 +805,9 @@ func (m *VolumeManager) discoverDataOnDrives() error {
 			}
 			continue
 		}
-		ll.Info(discoverResult.Message)
+
+		ll.Debug(discoverResult.Message)
```
Comment (safronovD) [inline pkg/node/volumemgr.go:809]: Do we really need to have this message? Isn't one on the line 802 enough?

Comment (AnnaDutova) [reply pkg/node/volumemgr.go:809]: I have never seen log from the line 802

**Review by mishoyama — APPROVED** [review]

**Review by safronovD — APPROVED** [review]

**Review by misanthropicat — APPROVED** [review]

**Review by yurkov-anton — APPROVED** [review]

**Pull request merged**